src/v2/coins/abstractUtxoCoin.js

```diff
@@ -1369,4 +1393,19 @@ AbstractUtxoCoin.AddressTypes = Object.freeze({
   P2WSH: 'p2wsh'
 });
 
+AbstractUtxoCoin.AddressTypeChains = Object.freeze({
+  p2sh: {
+    main: 0,
+    change: 1
+  },
+  'p2sh-p2wsh': {
```

Comment:
this is in quotes because of the dash in the middle. other ones are not in quotes because Lint enforces it

Reply:
Yep, make sense